### DISCUSSION TITLE
go through major packages that were built using interlisp

### DISCUSSION THREAD

**Discussion by masinter** [Interlisp/medley#8] [Ideas]
loops, trillium, notecards where there are sources and some docs or understanding, load, test out, document with video demo.

Recording zoom calls of yourself and a screen share is how i understand people are making videos, save in the cloud and have someone edit it.

**Comment by masinter** [comment]
From a discussion of the PARC to CHM.
What has to happen with Notecards and Rooms? @Anzus is working on getting notecards recompiled and making a video demo.(also has Medley 2.0 running under DOS).
 If there were a lot of people involved in Notecards trials, maybe some of them would like to participate? Or help?
New separate issues should be written to attract new helpers, of course.  

On Mon, Aug 10, 2020 at 12:30 PM Nick Briggs <nicholas.h.briggs@gmail.com> wrote:
Scanning the file names (but not the contents) I don't see anything that shouldn't be public.

I see a lot of names of people I recognize and hadn't thought about in a while, though, particularly in the 1.2k/notefile data...
Tadd Hogg, John Tang, Stu Card, Peggy Irish, Susan Newman, Kris Halvorsen, Marikka Rypa, Jeff Shrager, Carlin De Cato, Steve Bagley, Ken Feuerman, John Barwise, Lucy Suchman, Julian Orr, Frank Halasz, Dan Russell, Annie Zaenen, Richard Burton, Pete Pirolli, John Barwise, Randy Trigg, David Levy.   What a cast of characters.

I think the trailing letter on the version number matched up with the Interlisp version it ran in (k = Koto, i = Intermezzo, m = Medley), and what I'm not seeing is a Medley compatible sysout -- the most recent sysout is a Koto one.  There is the 1.3m directory, and it might be possible to load Notecards in Medley from source.

**Comment by Anzus** [comment]
As of this evening I’ve gotten NoteCards to compile and load. I’ve been able to load a couple of my old notefiles, but the really big one I have fails on FONTCREATE, attempting to create the font for (GACHA 8 (MEDIUM REGULAR REGULAR) 9 PRESS).

I’ve got PRESSFONTWIDTHSFILES pointing to the FONTS.WIDTHS, which I thought is what it takes for FONTCREATE to build the right Press font? 

Arun Welch
welch@anzus.com

**Reply by masinter** [reply]
@Anzus how about adding a notecards folder with sources, and a README that says how to compile and load.
And some example notefiles?

**Comment by Anzus** [comment]
Hmm. I’ve got DISPLAYFONTDIRECTORIES set to the top of fonts/displayfonts, which seems to have 104 subdirectories. Is that sufficient, or do I need to set it to the full list?

Arun Welch
welch@anzus.com

**Comment by masinter** [comment]
make sure you are starting with the head of the master branch of interlisp medley, and the font file names are ALL UPPERCASE.
(I made a mistake of thinking filename case didn't matter but it does because the font code does a DIRECTORY and checks for EQ of interned strings.

**Comment by Anzus** [comment]
Ahhh. Looks like they’re all lowercase in the repo on github.

Arun Welch
welch@anzus.com
